dockers/docker-sonic-telemetry/telemetry.sh

```diff
@@ -36,7 +36,7 @@ elif [ -n "$X509" ]; then
     SERVER_CRT=$(echo $X509 | jq -r '.server_crt')
```

Comment:
In my opinion we can remove X509 condition(line 35 elif). We have repalced that with TELEMETRY|Certs and TELEMETRY|gnmi config. If no one has any objection I can clean that up